### PR TITLE
More decimal fixes

### PR DESCRIPTION
**Pull request by saeta-eth**

### CODE REVIEW

File: src/hooks/aelin/history/useAelinDealsSponsored.tsx [hunk 1]
```diff
@@ -35,22 +35,33 @@ export async function fetcherDealsSponsored(variables: DealSponsoredsQueryVariab
...
             const amountEarned = formatToken(
               dealSponsored.amountEarned,
-              dealSponsored.pool.deal?.underlyingDealTokenDecimals || 0,
+              dealSponsored.pool.deal?.underlyingDealTokenDecimals ||
+                dealSponsored.pool.upfrontDeal?.underlyingDealTokenDecimals ||
+                BASE_DECIMALS,
+              DISPLAY_DECIMALS,
             )
 
-            const sponsorFee = formatToken(dealSponsored.pool.sponsorFee, BASE_DECIMALS)
```
Comment (saeta-eth) [inline src/hooks/aelin/history/useAelinDealsSponsored.tsx:53]: wrong decimals

File: src/utils/aelinPoolUtils.ts [hunk 2]
```diff
@@ -80,15 +80,25 @@ export function getPurchaseTokenCap<
...
+    ),
   }
 }
 
 // returns the sponsor's fee amount (max is 98%)
-export function getSponsorFee<P extends { sponsorFee: string }>(pool: P) {
+export function getSponsorFee(pool: PoolCreated) {
   return {
     raw: BigNumber.from(pool.sponsorFee),
-    formatted: `${formatToken(pool.sponsorFee, BASE_DECIMALS, 2)}%`,
```
Comment (saeta-eth) [inline src/utils/aelinPoolUtils.ts:91]: wrong decimal

File: src/utils/aelinPoolUtils.ts [hunk 3]
```diff
@@ -394,20 +404,27 @@ export function getInvestmentDealToken(
...
+      collectionRule.purchaseAmount,
+      pool.purchaseTokenDecimals || BASE_DECIMALS,
+      true,
+    )
+
     return {
       ...collectionRule,
       purchaseAmount: {
         raw: purchaseAmountBN.toBN(),
-        formatted: formatToken(purchaseAmountBN.toBN(), BASE_DECIMALS, 3),
```
Comment (saeta-eth) [inline src/utils/aelinPoolUtils.ts:410]: wrong decimal

File: src/hooks/aelin/history/useAelinDealsFunded.tsx [hunk 4]
```diff
@@ -35,14 +36,14 @@ export async function fetcherDealsFunded(variables: DealFundedsQueryVariables) {
               dealFunded.amountFunded,
               dealFunded.pool.deal?.underlyingDealTokenDecimals ||
                 dealFunded.pool.upfrontDeal?.underlyingDealTokenDecimals ||
-                0,
-              10,
+                18,
```
Comment (0xDmitry) [inline src/hooks/aelin/history/useAelinDealsFunded.tsx:39]: BASE_DECIMALS here?

Comment (saeta-eth) [reply src/hooks/aelin/history/useAelinDealsFunded.tsx:39]: Yep, already updated!

**Review by 0xDmitry — APPROVED** [review]

**Pull request merged**